Mergin/plugin.py

```diff
@@ -535,7 +535,7 @@ def remove_local_project(self):
                 mp.log.removeHandler(log_file_handler)
                 del mp
 
-                shutil.rmtree(self.path)
+                QTimer.singleShot(250, lambda: shutil.rmtree(self.path))
```

Comment:
can we add comment to code why QTimer is added?

Reply:
Done